app/src/pages/liquidity/liquidity.tsx

```diff
@@ -1,6 +1,5 @@
 import React, { useEffect, useState } from 'react';
```

Comment:
I had to add a line here to get this to work:
import Typography from '@material-ui/core/Typography';

Reply:
2023-01-23T19:20:36.369Z [INFO]: [eslint]
                                 src/pages/liquidity/liquidity.tsx
                                 Line 81:8:  'Typography' is not defined  react/jsx-no-undef
                                 Line 84:8:  'Typography' is not defined  react/jsx-no-undef
                                 Search for the keywords to learn more about each error.


Build is failing for that reason.